### PR TITLE
Fiber 💨 v0.8.1

### PR DESCRIPTION
**Pull request by Fenny**
Bumped to `v0.6.1`
```
Auditing /FrameworkBenchmarks/frameworks/Go/fiber:
No problems to report
```
```
| fiber
|       plaintext     : PASS
|       query         : PASS
|       json          : PASS
|       db            : PASS
|       update        : PASS
```
Thanks, @nbrady-techempower  :heart:

### CODE REVIEW

File: frameworks/Go/fiber/benchmark_config.json [hunk 1]
```diff
@@ -3,12 +3,15 @@
...
+      "query_url": "/queries?queries=",
+      "update_url": "/update?queries=",
       "plaintext_url": "/plaintext",
       "port": 8080,
       "approach": "Realistic",
       "classification": "Platform",
-      "database": "None",
-      "framework": "Fiber",
+      "database": "MySQL",
+      "framework": "None",
```
Comment (nbrady-techempower) [inline frameworks/Go/fiber/benchmark_config.json]: The rest of the PR looks good. If you could just switch this back to `Fiber` I can merge this in.

Comment (Fenny) [reply frameworks/Go/fiber/benchmark_config.json]: Gotcha, thanks :+1:

**Comment by Fenny** [comment]
Travis is having some mysql problems?

**Comment by nbrady-techempower** [comment]
@Fenny This isn't you. I'll be investigating shortly.

**Comment by nbrady-techempower** [comment]
@Fenny could you merge in the latest master? it has the fix for mysql

**Comment by Fenny** [comment]
Just missed the round :disappointed:

File: frameworks/Go/fiber/src/server.go [hunk 2]
```diff
@@ -1,19 +1,116 @@
...
+	stmtFortuneSelect *sql.Stmt
+)
+
+// JSON struct for json serialization
 type JSON struct {
 	Message string `json:"message"`
 }
 
+var hwJSON = JSON{"Hello, World!"}
+var hwTEXT = "Hello, World"
```
Comment (nbrady-techempower) [inline frameworks/Go/fiber/src/server.go]: Your response is missing the `!`

File: frameworks/Go/fiber/src/server.go [hunk 3]
```diff
@@ -1,19 +1,116 @@
...
+	stmtWorldUpdate   *sql.Stmt
+	stmtFortuneSelect *sql.Stmt
+)
+
+// JSON struct for json serialization
 type JSON struct {
 	Message string `json:"message"`
 }
 
+var hwJSON = JSON{"Hello, World!"}
```
Comment (nbrady-techempower) [inline frameworks/Go/fiber/src/server.go]: This can't be declared outside of the `jsonSerialization` function. It must happen on every request.

Comment (Fenny) [reply frameworks/Go/fiber/src/server.go]: The serialization still happens in the request, but I changed it it the new pull to be sure.

File: frameworks/Go/fiber/src/server.go (hunk 2 above)
Comment (Fenny) [reply frameworks/Go/fiber/src/server.go]: Sorry my bad.

**Pull request merged**